.github/workflows/main.yml

```diff
@@ -388,6 +388,24 @@ jobs:
             build-type: Debug
             arch: x64
             toolchain: ""
+          - os: windows-2022
+            build-type: Release
+            arch: ARM64
+            toolchain: ""
+          - os: windows-2022
+            build-type: Debug
+            arch: ARM64
+            toolchain: ""
+          - os: windows-2022
+            build-type: Release
+            arch: ARM64EC
+            toolchain: ""
+            extra-cmake-flags: -DCMAKE_SYSTEM_VERSION="10.0.22621.0"
```

Comment:
Using the Windows 11 SDK is required for CMake to build ARM64EC correctly: https://stackoverflow.com/a/77651884